var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -44,29 +48,24 @@ class PyPybind11(CMakePackage, PythonPackage):
     version("2.1.1", sha256="f2c6874f1ea5b4ad4ffffe352413f7d2cd1a49f9050940805c2a082348621540")
     version("2.1.0", sha256="2860f2b8d0c9f65f0698289a161385f59d099b7ead1bf64e8993c486f2b93ee0")
 
-    depends_on("ninja", type="build")
     depends_on("py-setuptools@42:", type="build")
     depends_on("py-pytest", type="test")
     depends_on("python@2.7:2.8,3.5:", type=("build", "run"))
-    depends_on("cmake@3.13:", type="build")
-    depends_on("cmake@3.18:", type="build", when="@2.6.0:")
+
+    depends_on("py-pip", type="build")
+    depends_on("py-wheel", type="build")
+    extends("python")
+
+    with when("buildsystem=cmake"):
```

Comment:
Is this ever not the case?

Reply:
Eheh, this is not the case if you inherit from 2 build-systems :slightly_smiling_face:

Reply:
What I meant to say: in this case we can remove it, but since `ninja` and `cmake` are likely to be related to the CMake build-system it is safe to protect these with `buildsystem=cmake`.